kuma/payments/tests/test_views.py

```diff
@@ -29,6 +29,15 @@ def test_payments_index(client):
     assert response.status_code == 200
 
 
+@pytest.mark.django_db
+def test_payments_index_email_renders(client):
+    """Subscriptions email renders on payments landing page"""
+    response = client.get(reverse("payments_index"))
+    email_link = "mailto:" + settings.CONTRIBUTION_SUPPORT_EMAIL
+    content = response.content.decode(response.charset)
+    assert email_link in content
```

Comment:
Actually, what I had in my branch, that I'm going to discard in favor of yours, was something like this:

```diff
diff --git a/kuma/payments/tests/test_views.py b/kuma/payments/tests/test_views.py
index bfec12dd4..4fce81e06 100644
--- a/kuma/payments/tests/test_views.py
+++ b/kuma/payments/tests/test_views.py
@@ -4,6 +4,7 @@ from unittest import mock
 import pytest
 import stripe
 from django.conf import settings
+from pyquery import PyQuery as pq
 from waffle.testutils import override_flag
 
 from kuma.core.tests import assert_no_cache_header, assert_redirect_to_wiki
@@ -28,6 +29,13 @@ def test_payments_index(client):
     response = client.get(reverse("payments_index"))
     assert response.status_code == 200
 
+    doc = pq(response.content)
+    assert (
+        doc.find('.faq a[href^="mailto:{settings.CONTRIBUTION_SUPPORT_EMAIL}"]').text()
+        == settings.CONTRIBUTION_SUPPORT_EMAIL
+    )
+    assert doc.find("h1").text() == "Become a monthly supporter"
+
 
 @pytest.mark.django_db
 @override_flag("subscription", True)

```

At first, I added `assert settings.CONTRIBUTION_SUPPORT_EMAIL in html`
but that worked before I had even started doing anything. (It's because the email gets part of the Jinja template AFTER SSR does its thing)
Also, the same with `assert "Become a monthly supporter" in html` which also suspiciously worked because it's part of the `<title>`. 

The above suggested diff is work in progress. I hope you can weave it in.

Reply:
Thanks Peter! I added your test and changed a line to `assert settings.CONTRIBUTION_SUPPORT_EMAIL in doc.find(".contributions-page").text()`